scripts/test_priority_plugin.sh

```diff
@@ -66,6 +73,51 @@ function start_container() {
 	done
 }
 
+#!/bin/bash
+
+function compile_priority_plugin() {
+    if [ "$RELEASE_MODE" = true ]; then
+        cargo build --bin auditor-priority-plugin --target x86_64-unknown-linux-musl --release
+    else
+        cargo build --bin auditor-priority-plugin --target x86_64-unknown-linux-musl
+    fi
+}
+
+function start_priority_plugin() {
+    if [[ -z "${SKIP_COMPILATION}" ]]; then
+        compile_priority_plugin
+    fi
+
+    if [ "$RELEASE_MODE" = true ]; then
+        local config_file="$1"
+        docker exec "${COMPOSE_PROJECT_NAME}-slurm-1" /auditor-priority-plugin /$config_file &
+    else
+        local config_file="$1"
+        docker exec "${COMPOSE_PROJECT_NAME}-slurm-1" /auditor-priority-plugin /$config_file &
+    fi
```

Comment:
Here we don't need the disambiguation between debug and release mode, because the path is the same inside the docker container